rclcpp/src/rclcpp/node_interfaces/node_graph.cpp

```diff
@@ -28,6 +28,12 @@
 using rclcpp::node_interfaces::NodeGraph;
 using rclcpp::exceptions::throw_from_rcl_error;
 using rclcpp::graph_listener::GraphListener;
+using rcl_get_info_by_topic_func_t = rcl_ret_t (* )(
+  const rcl_node_t *,
+  rcutils_allocator_t *,
+  const char *,
+  bool,
+  rmw_topic_endpoint_info_array_t *);
```

Comment:
This shouldn't be needed. Please see https://github.com/Barry-Xu-2018/rclcpp/pull/2.